src/api/services/ListingItemService.ts

```diff
@@ -101,9 +101,12 @@ export class ListingItemService {
         // this.log.debug('listingItemObjects to save: ', JSON.stringify(listingItemObjects, null, 2));
 
         // If the request body was valid we will create the listingItem
-        const listingItem: any = await this.listingItemRepo.create(body)
+        let listingItem;
+        await this.listingItemRepo.create(body)
             .catch(reason => {
                 this.log.error('ERROR: ', reason);
+            }).then(res => {
+                listingItem = res;
```

Comment:
is there a reason to always use then()... and 6 lines when we can do the same in one line? 

const listingItem: any = await this.listingItemRepo.create(body).catch(if needed, most of the times we dont need to catch the exception and just let it pass forward);

I'm fine with changing ways, if we have reasons, but this isnt as readable code as just one line. And producing similar code, always in a similar way, and as readable, is something we should try to do. If we change some way we do something, we should try to refactor the code and do it similarly elsewhere too.

Right now, I think, if there's an error creating the listingItem. Instead of error getting thrown instantly, because of the catch, I think you are 'silently' catching the thrown error (you are not throwing another error, just logging it) and then continuing to save iteminformation which will fail (because of undefined listing_item_id) and then it will throw another error, which is not the real cause of the problem.